LocalPayment/src/main/java/com/braintreepayments/api/localpayment/LocalPaymentRequest.kt

```diff
@@ -0,0 +1,171 @@
+ *
+ * @see [User Data policies for the Google Play Developer Program ](https://support.google.com/googleplay/android-developer/answer/10144311.personal-sensitive)
+ *
+ * @see [Examples of prominent in-app disclosures](https://support.google.com/googleplay/android-developer/answer/9799150?hl=en.Prominent%20in-app%20disclosure)
+ */(private val hasUserLocationConsent: Boolean) {
+    /**
+     * @param address Optional - The address of the customer. An error will occur if this address is not valid.
+     */
+    @JvmField
+    var address: PostalAddress? = null
+
+    /**
+     * @param amount Optional - The amount for the transaction.
+     */
+    @JvmField
+    var amount: String? = null
+
+    /**
+     * @param bankIdentificationCode Optional - the Bank Identification
+     * Code of the customer (specific to iDEAL transactions).
+     */
+    var bic: String? = null
+
+    /**
+     * @param currencyCode Optional - A valid ISO currency code to use for the transaction. Defaults to merchant
+     * currency code if not set.
+     */
+    @JvmField
+    var currencyCode: String? = null
+
+    /**
+     * @param displayName Optional - The merchant name displayed inside of the Local Payment flow
+     * when starting the payment.
+     */
+    @JvmField
+    var displayName: String? = null
+
+    /**
+     * @param email Optional - Payer email of the customer.
+     */
+    @JvmField
+    var email: String? = null
+
+    /**
+     * @param givenName Optional - Given (first) name of the customer.
+     */
+    @JvmField
+    var givenName: String? = null
+
+    /**
+     * @param merchantAccountId Optional - A non-default merchant account to use for tokenization.
+     */
+    @JvmField
+    var merchantAccountId: String? = null
+
+    /**
+     * @param paymentType - The type of payment
+     */
+    @JvmField
+    var paymentType: String? = null
+
+    /**
+     * @param paymentTypeCountryCode The country code of the local payment. This value must be one of
+     * the supported country codes for a given local payment type listed.
+     * For local payments supported in multiple countries, this value
+     * may determine which banks are presented to the customer.
+     * @see [Supported Country Codes](https://developer.paypal.com/braintree/docs/guides/local-payment-methods/client-side-custom/android/v4.invoke-payment-flow)
+     */
+    @JvmField
+    var paymentTypeCountryCode: String? = null
+
+    /**
+     * @param phone Optional - Phone number of the customer.
+     */
+    @JvmField
+    var phone: String? = null
+
+    /**
+     * @param shippingAddressRequired - Indicates whether or not the payment needs to be shipped. For digital goods,
+     * this should be false. Defaults to false.
+     */
+    var isShippingAddressRequired: Boolean = false
+
+    /**
+     * @param surname Optional - Surname (last name) of the customer.
+     */
+    @JvmField
+    var surname: String? = null
+
+    fun hasUserLocationConsent(): Boolean {
```

Comment:
We should be able to remove this getter function. It'll be a breaking change that we can call out in the changelog. 

similar change: https://github.com/braintree/braintree_android/blob/474febc875cac373c3d6823b59903bb8e674dd41/CHANGELOG.md?plain=1#L13